src/_guides/language/language-tour.md

```diff
@@ -2999,6 +3002,60 @@ class Employee extends Person {
   example, arguments can call static methods but not instance methods.
 {{site.alert.end}}
 
+<a name="super-parameters"></a>
+To avoid having to manually pass each parameter
+into the super invocation of a non-redirecting constructor,
+you can use super-initializer parameters to forward parameters
+to the specified or default superclass constructor. 
+Super-initializer parameters have similar syntax and semantics to
+[initializing formal parameters](#initializing-formal-parameters):
```

Comment:
My understanding is that super parameters can't be when the *current* constructor is a redirecting constructor, but when I read this I'm not sure we're talking about the superclass's constructor. 

I think it might be better to say this:

```suggestion
To avoid having to manually pass each parameter
into the super invocation of a constructor,
you can use super-initializer parameters to forward parameters
to the specified or default superclass constructor.
This feature can't be used with redirecting constructors.
Super-initializer parameters have similar syntax and semantics to
[initializing formal parameters](#initializing-formal-parameters):
```

For example, this is allowed:

```dart
class Vector2d {
  final double x;
  final double y;
  Vector2d(double x, double y) : this._(x,y); // redirecting constructor
  Vector2d._(this.x, this.y);
}

class Vector3d extends Vector2d {
  final double z;
  Vector3d(super.x, super.y, this.z);
}
```

Reply:
+1

Reply:
Looks good to me, thanks for clearing this up :)